Components/Input/input.css

```diff
@@ -0,0 +1,78 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
```

Comment:
don't repeat yourself. move this to a separate file

Reply:
Ok